CASMHMS-5690 Update HMS test known issues section with ComponentEndpoint EthernetInterfaces failure and remediation csm-1.2 (#2352)
Cray-HPE/docs-csm: troubleshooting/interpreting_hms_health_check_results.md

The following types of HMS test failures should **not** be considered blocking for system installations:

> - Failures due to hardware issues on individual nodes (alerts or warning flags set in HSM)

Reviewer: ```suggestion
- Failures because of hardware issues on individual nodes (alerts or warning flags set in HSM)
```

Failures of this test caused by `AuxiliaryController` endpoints for Cassini mezzanine cards can be safely ignored.

### `ComponentEndpoints` with `EthernetInterfaces` named `DE*` in HSM

> The following HMS functional test may fail due to a known issue because of `ComponentEndpoints` with `EthernetInterfaces` named `DE*` in HSM:

Reviewer: ```suggestion
There is a known issue that causes the HMS functional test to fail when HSM `ComponentEndpoints` have `EthernetInterfaces` with names of the format `DE*`.
```

- `test_smd_component_endpoints_ncn-functional_remote-functional.tavern.yaml`

This issue looks similar to the following in the test output:

```text
        Traceback (most recent call last):
          File "/usr/lib/python3.8/site-packages/tavern/schemas/files.py", line 106, in verify_generic
            verifier.validate()
          File "/usr/lib/python3.8/site-packages/pykwalify/core.py", line 166, in validate
            raise SchemaError(u"Schema validation failed:\n - {error_msg}.".format(
        pykwalify.errors.SchemaError: <SchemaError: error code 2: Schema validation failed:
         - Value 'DE07A000' does not match pattern '^$|[0-9]+|HPCNet[0-9]+|ManagementEthernet'. Path: '/ComponentEndpoints/34/RedfishSystemInfo/EthernetNICInfo/0/RedfishId'.
         - Value 'DE07A001' does not match pattern '^$|[0-9]+|HPCNet[0-9]+|ManagementEthernet'. Path: '/ComponentEndpoints/34/RedfishSystemInfo/EthernetNICInfo/1/RedfishId'.: Path: '/'>
```

Failures of this test caused by `EthernetInterface` IDs of the form `DE*` can be safely ignored.

> This issue may be remediated by rediscovering the BMCs with `EthernetInterfaces` named `DE*`.

Reviewer: ```suggestion
This issue may be remediated by rediscovering the BMCs associated with these `EthernetInterfaces`.
```
